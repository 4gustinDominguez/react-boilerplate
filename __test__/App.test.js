import React from 'react';
import { shallow } from 'enzyme';
import { App } from '../src/components/App';

describe('App component', () => {
  it('should render correctly in mode', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
