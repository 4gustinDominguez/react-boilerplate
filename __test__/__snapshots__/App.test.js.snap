// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly in mode 1`] = `ShallowWrapper {}`;
